src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -694,7 +732,7 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
         case _ => Nil
       }
       val dependencies = upstream.flatten ++ downstream
-      val dependenciesResolved = dependencies.isEmpty || dependencies.forall(isDone)
+      val dependenciesResolved = dependencies.isEmpty || dependencies.forall(d => isDone(d) || isPreempted(d))
```

Comment:
Also, as a sidenote on this line, I'm pretty sure `forall` covers the `.isEmpty` case already